Source/Core/DolphinWX/Debugger/CodeView.cpp

```diff
@@ -196,34 +196,10 @@ u32 CCodeView::AddrToBranch(u32 addr)
 
 void CCodeView::InsertBlrNop(int Blr)
```

Comment:
Could you fix that parameter name to conform to our coding style while you're at it?